docs/documentation/stories/configure-hmr.md

```diff
@@ -1,4 +1,4 @@
-**Documentation below is deprecated and we no longer accept PRs to improve this. The new documentation is available [here](https://angular.io/guide/build)**.
+**HMR is a WebPack feature that is not officially supported by Angular. Community contributors can submit PRs against this page to add corrections, new information, or advice.**.
```

Comment:
Can you kindly squash the commits and rebase to make this green?

 Also, WebPack should be Webpack

Reply:
Apparently I can't squash, and in attempting to rebase, I might have corrupted this branch altogether. But I have used the GitHub editor to change to "webpack", which is how they use the term on their own site.